package.json

```diff
@@ -526,9 +526,9 @@
     "open-in-browser": "vscode-test-web --extensionDevelopmentPath=. .",
     "lint": "eslint . --ext .ts",
     "lint-fix": "eslint . --ext .ts --fix",
-    "format": "prettier --write '**/*.ts'",
-    "format-check": "prettier --check '**/*.ts'",
-    "test": "NODE_OPTIONS=\"--experimental-vm-modules\" jest",
+    "format": "prettier --write \"**/*.ts\"",
+    "format-check": "prettier --check \"**/*.ts\"",
+    "test": "cross-env NODE_OPTIONS=\"--experimental-vm-modules\" jest",
```

Comment:
Just to confirm, this won't break non-Windows (MacOS, Linux) environments, right?

Reply:
Correct it won't break non-Windows! I tested it on my Mac and it worked and I put the screenshots of the tests in the PR description.